src/meta/src/hummock/manager/compaction_group_manager.rs

```diff
@@ -562,6 +548,7 @@ impl HummockManager {
             }
         }
 
+        drop(versioning_guard);
```

Comment:
Why was this drop moved down a few lines?

Reply:
Because we are still using `versioning` in `let need_cancel = HummockManager::is_compact_task_expired(task, &versioning.current_version);`, which holds reference to `versioning_guard`.